src/commands/domain.js

```diff
@@ -26,17 +26,29 @@ function getFavouriteDomain ({ ownerId, appId }) {
 }
 
 async function list (params) {
-  const { alias } = params.options;
+  const { alias, format } = params.options;
   const { ownerId, appId } = await AppConfig.getAppDetails({ alias });
 
   const app = await getApp({ id: ownerId, appId }).then(sendToApi);
   const favouriteDomain = await getFavouriteDomain({ ownerId, appId });
-  return app.vhosts.forEach(({ fqdn }) => {
-    const prefix = (fqdn === favouriteDomain)
-      ? '* '
-      : '  ';
-    Logger.println(prefix + fqdn);
-  });
+
+  const formattedDomains = app.vhosts.map(({ fqdn }) => ({
+    fqdn,
+    favourite: fqdn === favouriteDomain,
+  }));
+
+  switch (format) {
+    case 'json': {
+      Logger.println(formattedDomains);
+      break;
+    }
+    case 'human':
+    default: {
+      formattedDomains.forEach(({ fqdn, favourite }) => {
+        Logger.println(`${favourite ? '*' : ' '} ${fqdn}`);
```

Comment:
By default, this leads to ' domain.cleverapps.io' with an empty space at the beginning. I suggest to replace it with:

```suggestion
        Logger.println(`${fqdn}${favourite ? color.green(' (favourite)') : ''} `);
```

(it will need `const color = require('colors/safe');`)